frontend/src/main/java/com/google/tripmeout/frontend/service/PlacesApiPlaceService.java

```diff
@@ -0,0 +1,35 @@
+package com.google.tripmeout.frontend.service;
+
+import com.google.inject.Inject;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+import com.google.maps.GeoApiContext;
+import com.google.maps.PlaceDetailsRequest;
+import com.google.maps.errors.ApiException;
+import com.google.maps.errors.InvalidRequestException;
+import com.google.maps.model.PlaceDetails;
+import com.google.tripmeout.frontend.PlaceVisitModel;
+import java.io.IOException;
+import java.lang.InterruptedException;
+
+@Singleton
```

Comment:
Note: this service does not technically require a singleton binding. You can imagine scenarios where you might want multiple place services, e.g., initialized with different API keys (contexts).

Instead, the singleton scope should be added to the module binding.